opta/nice_subprocess.py

```diff
@@ -22,79 +24,133 @@ def nice_run(  # type: ignore # nosec
     capture_output: bool = False,
     timeout: Optional[float] = None,
     exit_timeout: Optional[float] = None,
+    tee: Optional[bool] = True,
```

Comment:
```suggestion
    tee: bool = True,
```

- `Optional[T]` means the value can be `None`, which is not what we want in this case.
- Should this default to `False` instead to preserve the existing behavior? That would mean that this PR wouldn't need to update every caller.